web/models.py

```
@@ -50,6 +50,10 @@ def __str__(self):
 
 class Profile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
+    points = models.PositiveIntegerField(
+        default=0,
+        help_text="User points balance"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)